packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -237,10 +496,25 @@ it('should render tick marks', () => {
     />,
   );
   assertBaseElement(container);
+  // TODO: Do we also need to check the content of the slider ticks?
+  expect(container.querySelectorAll('.iui-slider-tick').length).toBe(5);
+});
+
+it('should render tick marks (vertical)', () => {
```

Comment:
Not needed as it is not vertical orientation specific and is already tested.

Reply:
**[DONE]** Thanks, removed this test